src/libstd/sys/common/unwind/mod.rs

```diff
@@ -172,7 +172,7 @@ pub fn panicking() -> bool {
 #[inline(never)]
 #[no_mangle]
 #[allow(private_no_mangle_fns)]
```

Comment:
You're right, still required: http://is.gd/Z4qyRE

On Fri, Dec 25, 2015 at 8:37 PM, Steven Fackler <notifications@github.com>
wrote:

> In src/libstd/sys/common/unwind/mod.rs
> <https://github.com/rust-lang/rust/pull/30557#discussion_r48444144>:
>
> > @@ -172,7 +172,7 @@ pub fn panicking() -> bool {
> >  #[inline(never)]
> >  #[no_mangle]
> >  #[allow(private_no_mangle_fns)]
>
> Possibly - not sure how deeply that lint looks. This function is now
> public within its module, but not public outside of the crate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/30557/files#r48444144>.
>